### ISSUE TITLE
UTF8 support

### ISSUE THREAD

**Issue by xardbaiz** [marc0l92/joplin-plugin-plantUML#4]
Please add support of SVG render with UTF8 symbols or force convert to UTF8 for non-Latin characters

In this case, I'm trying to use the Russian language
```plantuml
@startmindmap
* Conditional cases
** " Если вы не любите котов, то вы просто не умеете их готовить " \n General knowledges / always true facts.
*** [[joplin://6f0b3c7b49ae4d238e804c23a2e84dc7 Zero conditional]]
** Possible future
*** [[joplin://cc742d27ec6f4cf29978c4dfced0c8b) First conditional]]
@endmindmap
```

Joplin:
![image](https://user-images.githubusercontent.com/92844117/145452486-87f416e3-751b-4666-bb84-1d2d6c7102bb.png)

Example in PlantUML Web Server UI [_(design link)_](https://www.plantuml.com/plantuml/uml/NL31IiD04BttAmPl1h4bNPhKKq7GgsTh3vlPYKItcvBTKBnPnPFYpG-nKYXMwo_C_f5POX5wshUFUM_clP5ridBrL7TQD2sCcifP9uBJqgZ25QMH6b1QifqeWWFWDx_W3Q-1b_uP-9jNm1l_ok-yzW_XyydR1cnvUGJX_PlxyVSDlmXCA_AF_CKhNhMYjNy2_jWfWvL_QPPT6JWdGvNKS6lAkIPrJHQw8FLStbbmLKsGItJsk3akWl7uffpfmfnqkuEyb_KnoSHGab1nFwMq9p3kovXIeJ21IwfAmF-Caqd8Tr5QMsIQ8AzTNT6-As8YOXKdX8DSO1uFXqcAGkL8geTfTWXdHMNTdks8ZDhL6d1JzYy0) :
![image](https://user-images.githubusercontent.com/92844117/145452573-852a0040-df58-4604-b0e9-9c2f03dac73d.png)

**Comment by marc0l92** [comment]
Thanks for creating this issue.

You can find the fix in the version 1.7.0 of this plugin.
Let me know if it works as expected :)

**Comment by xardbaiz** [comment]
Thanks. It's working now
